ga/latest/kernel/helpers/build/features-installed.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Suggest to move this to helpers/build/internal directory - just to separate the internal scripts from user scripts

Reply:
Moved this script and references to the helpers/build/internal directory